### ISSUE TITLE
Help to understand particles behavior when interacting with rigidbody

### ISSUE THREAD

**Issue by intee25** [InteractiveComputerGraphics/SPlisHSPlasH#168]
Hello,

For my application, I am trying to instantiate fluid particles in a mesh model of a jug. I am creating a fluid block on top of the jug and trying to run the simulation to have the particles collect inside the jug. However, the particles seem to be hitting an imaginary wall(?) outside the jug itself. 
I am attaching the snapshots along with the .OBJ file of the jug and configuration+code of the environment used. 
Any help rectifying this issue is highly appreciated. Thank you!
![Screenshot from 2022-04-14 23-51-11](https://user-images.githubusercontent.com/88543337/163483059-7b8f8bbd-48c8-46bc-9577-3680fcf52fff.png)
![Screenshot from 2022-04-14 23-51-33](https://user-images.githubusercontent.com/88543337/163483062-1a8ba76f-a05a-4f33-af3a-7d4a36df8368.png)
![Screenshot from 2022-04-14 23-52-05](https://user-images.githubusercontent.com/88543337/163483071-a79751db-7059-4f87-afe3-dc5be424275f.png)

```
"Configuration": 
	{
		"pause":true,
               "particleRadius": 0.003,
		"timeStepSize": 0.0001,
                 "cameraPosition":[0,0.5,1],
		"density0": 1000, 
		"simulationMethod": 4,
		"gravitation": [0,-9.81,0], 
		"cflMethod": 0, 
		"cflFactor": 0.5
	}
```

```
scene.fluidBlocks.append(
        Scenes.FluidBlock(
            id='Fluid', 
            box=Scenes.Box([-0.025, 0.22, -0.025], [.025, .32, .025]), 
            mode=0, 
            initialVelocity=[0.0, 0.0, 0.0],
            ))
scene.boundaryModels.append(Scenes.BoundaryData(
        meshFile="../models/elongated_new.obj", 
        translation=[0, 0, 0], scale=[1,1,1], color=[0.5, 0.5, 0.5, 1.0], isAnimated=True, 
        rotation=r,
        isWall=False, mapInvert=False, mapResolution=[25, 25, 25]))
```
[objfile.zip](https://github.com/InteractiveComputerGraphics/SPlisHSPlasH/files/8492493/objfile.zip)

**Comment by janbender** [comment]
That is strange. Can you please try to delete the cache files or start with the --no-cache option?

If this does not help, try to set the value "boundaryHandlingMethod" to 0.

Hope that helps.

**Comment by intee25** [comment]
Hello, I deleted the cache files and tried to run it, but the results remain the same.

It seems I am unable to change the boundary method. I am using the python interface and within the json file for the scene, I set "boundaryHandlingMethod": 0. However, in gui window, under the 'Simulation' tab, I still see 'Bender et al. 2019' instead of ''Akinci et al. 2012' for the boundary handling method. Are there additional things to be configured/coded in order to change the boundary method?
